src/components/sockend.js

```diff
@@ -88,27 +88,31 @@ module.exports = class Sockend extends Configurable(Component) {
                 name: 'sockendSub',
                 namespace: namespace,
                 key: originalKey,
-                subscribesTo: obj.advertisement.broadcasts,
+                subscribesTo: obj.advertisement.broadcasts
             }, discoveryOptions);
 
-            subscriber.onMonitorAdded = () => { };
+            subscriber.onMonitorAdded = () => {
+            };
 
             obj.subscriber = subscriber;
 
-            subscriber.on('**', function(data) {
+            subscriber.on('**', function (data) {
                 if (this.event == 'cote:added' || this.event == 'cote:removed') return;
 
                 let topic = this.event.split('::');
                 let namespace = '';
-
-                if (topic.length > 1) {
-                    namespace += '/' + topic[0];
-                    topic = topic.slice(1);
```

Comment:
why did you remove this? do namespaces work with this?

Reply:
might be a mistake, good catch let me check